fs/_ftp_parse.py

```diff
@@ -82,15 +85,14 @@ def parse_line(line):
 
 
 def _parse_time(t, formats):
-    t = " ".join(token.strip() for token in t.lower().split(" "))
-
     _t = None
```

Comment:
As the `if not _t` line has been removed, I guess this `_t = None` can be removed too?

Reply:
You're absolutely right, sorry.